spec/controllers/api/v1/tutorials_controller_spec.rb

```diff
@@ -34,8 +34,7 @@
       expect(assigns(:tutorial).description).to eql(attributes[:description])
       expect(assigns(:tutorial).heading).to eql(attributes[:heading])
       expect(assigns(:tutorial).tags.count).to eql(2)
-      expect(assigns(:tutorial).tags.first.name).to eql("cake")
-      expect(assigns(:tutorial).tags.last.name).to eql("cookie")
+      expect(assigns(:tutorial).tags.pluck(:name)).to match_array(['cake', 'cookie'])
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.